src/app/dashboard/executionsgrid/gridrow/gridrow.component.html

```diff
@@ -43,8 +43,15 @@
 
   <span class="grid-cell" *ngIf="dsExecution.executionProgress">
     <span *ngIf="dsExecution.executionProgress.currentPluginProgress">
-      {{ dsExecution.executionProgress.currentPluginProgress.processedRecords }} /
-      {{ dsExecution.executionProgress.currentPluginProgress.expectedRecords }}
+      <span *ngIf="dsExecution.executionProgress.currentPluginProgress.expectedRecords < 0">
+        {{ 'unknown' | translate }}
+      </span>
+
+      <span *ngIf="dsExecution.executionProgress.currentPluginProgress.expectedRecords > 0">
+        {{ dsExecution.executionProgress.currentPluginProgress.processedRecords }} /
+        {{ dsExecution.executionProgress.currentPluginProgress.expectedRecords }}
+      </span>
+
```

Comment:
What happens if the expectedRecords is exactly 0?

Reply:
Functionally this is the same as it was before, so the output would be "0".  The scope of this fix was to restore the "unknown" label.

![image](https://user-images.githubusercontent.com/4441899/55944836-bfb21d80-5c49-11e9-9ffc-d17e6f8961de.png)